bot/Exchanges/Binance.py

```diff
@@ -668,14 +687,13 @@ def updateSQLOrderModel(self, order, new_order_response, bot):
 		new_order_response. Should be part of the exchange interface  """
 
 		if order.is_test:
-			order.take_profit_price = self.toValidPrice(
-				symbol = order.symbol,
-				desired_price = Decimal(order.entry_price) * (Decimal(100) + Decimal(bot.profit_target))/Decimal(100), 
-				round_up=True)
+			order.entry_price = self.getCurrentTickPrice(order.symbol)['lastPrice']
```

Comment:
Why are we changing the entry_price here

Reply:
With fake market orders we have no way of knowing the price it sold for but getting the current tick price as price is not a parameter in the order but that part deserves some extra love and care. Don't think this works for all cases. Will look at it again and fix.